Ro the Bot/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "author": {
+        "name": "bakzkndd",
+        "id": "343103632453009408"
+    },
+    "service": "Ro the Bot",
+    "description": {
+        "en": "Ro the bot is the most diverse discord bot! Just imagine a function, and it probably has it."
+    },
+    "url": ["ro-the-bot.ga", "status.ro-the-bot.ga"],
+    "version": "1.0",
+    "logo": "https://m.imgur.com/gallery/YDmTRVG",
+    "thumbnail": "https://rothebot.is-inside.me/8gHKV9oG.png",

```

Comment:
For logo it should be an i.imgur url with the ending .png / .jpg
and upload the thumbnail picture also to imgur

Reply:
made it i.imgur for both